tests/unit/_internal/io/test_numpy.py

```diff
@@ -82,22 +82,20 @@ def test_numpy_openapi_responses():
 
 
 def test_verify_numpy_ndarray(caplog: LogCaptureFixture):
-    partial_check = partial(
-        from_example._verify_ndarray, exception_cls=BentoMLException  # type: ignore (test internal check)
-    )
+    partial_check = partial(from_example.validate_array, exception_cls=BentoMLException)
 
     with pytest.raises(BentoMLException) as ex:
         partial_check(np.array(["asdf"]))
-    assert f'Expecting ndarray of dtype "{from_example._dtype}"' in str(ex.value)  # type: ignore (testing message)
+    assert f'Expecting ndarray of dtype "{from_example._dtype}"' in str(ex.value)
 
     with pytest.raises(BentoMLException) as e:
         partial_check(np.array([[1]]))
-    assert f'Expecting ndarray of shape "{from_example._shape}"' in str(e.value)  # type: ignore (testing message)
+    assert f'Expecting ndarray of shape "{from_example._shape}"' in str(e.value)
 
-    # test cases whwere reshape is failed
+    # test cases where reshape is failed
     example = NumpyNdarray.from_sample(np.ones((2, 2, 3)))
-    example._enforce_shape = False  # type: ignore (test internal check)
-    example._enforce_dtype = False  # type: ignore (test internal check)
+    example._enforce_shape = False
```

Comment:
Do these not trigger type errors anymore?

Reply:
The issue with `type: ignore` is that it also silence editor linter. If you make a typo or access something that is not there, pyright linter won't pick up due to type ignore.

In this case I think it is ok to remove type ignore here since it is inside test.

Reply:
If we agree to ignore test types, then sure.

Reply:
I'm fine with ignoring this one error, but I still think typing in tests is also nice.